projects/packages/videopress/src/client/block-editor/blocks/video/index.ts

```diff
@@ -13,12 +13,14 @@ import transforms from './transforms';
 import videoPressBlockExampleImage from './videopress-block-example-image.jpg';
 import './style.scss';
 
-export const { name, title, description } = metadata;
+export const { name, title, description, attributes } = metadata;
 
 registerBlockType( name, {
 	edit: Edit,
+	title,
 	save,
 	icon,
+	attributes,
```

Comment:
@renatoagds, as we have a native-specific `index.ts` file now, we could remove these changes. I wanted to check with you before doing that. Is there any reason we should keep these changes for the web?

Reply:
We could keep it, no problem.